cockroachcloud/configure-cloud-org-sso.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Configure Cloud Organization Single Sign-On (SSO) (Preview)
+summary: Learn how to configure single sign-on (SSO) (Preview) for your CockroachDB Cloud organization.
+toc: true
+docs_area: manage
+---
+
+{% include_cached feature-phases/preview-opt-in.md %}
+
+{% include_cached cockroachcloud/sso-intro.md %}
+
+This page shows how to enable [Cloud Organization SSO](cloud-org-sso.html) and manage your configuration.
+
+## Plan to enable Cloud Organization SSO
+
+To ensure a smooth migration to Cloud Organization SSO, review the following information before you enable the feature.
+
+{{site.data.alerts.callout_info}}
+After it is enabled, Cloud Organization SSO cannot be disabled. To configure Cloud Organization SSO to behave like [Basic SSO](cloud-org-sso.html#basic-sso), refer to [Emulate Basic SSO](#emulate-basic-sso)
+{{site.data.alerts.end}}
+
+### Decide your custom URL
+
+When you [enable Cloud Organization SSO](#enable-cloud-organization-sso), you will configure a custom URL, that your users use to access {{ site.data.products.db }}. After the feature is enabled, your users must use that page to sign in. Your custom URL must be unique within {{ site.data.products.db }}. Communicate the custom URL to your users ahead of time.
+
+To change your custom URL after Cloud Organization SSO has been enabled, contact your account team.
+
+### Decide which authentication methods to enable
+
+All enabled authentication methods appear on your custom URL and are available to your users. By default, when you enable Cloud Organization SSO, the following authentication methods are enabled by default:
+
+- Email and password
+- Github
+- Google
+- Microsoft
+
+In addition, you can create authentication methods that connect to your identity provider using the [Security Access Markup Language (SAML)](https://en.wikipedia.org/wiki/Security_Assertion_Markup_Language) and [OpenID Connect (OIDC)](https://openid.net/connect/) identity protocols.
+
+Members are identified by their email address. To allow your members to select from multiple SSO authentication methods, ensure that the email addresses match across all of them. To allow members to migrate from password authentication to SSO, ensure that their email addresses match those in your identity providers.
```

Comment:
Not quite sure what "ensure that the email addresses match across all of them" means.

Reply:
If a user has a email/password account as abhinav@cockroachlabs.com, then when the admin enabled SSO using Okta, it must be ensured that it's the same email in there as well (and vice-versa) to allow for seamless switch between enabled methods. Otherwise if the email/password account is setup as abhinav@gmail.com, that would be treated as a different user altogether within the org.